### PR TITLE
fix: ValueError: Default token list has not been set.

### PR DESCRIPTION
**Pull request by sabotagebeats**
### What I did

fixes: #11 and #12 

### How I did it

updated black version, set converter to try except return false

### How to verify it

works now

### Checklist

- [x] Passes all linting checks (pre-commit and CI jobs)
- [ ] New test cases have been added and are passing
- [ ] Documentation has been updated
- [x] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Review by Ninjagod1251 — APPROVED** [review]

File: ape_tokens/converters.py [hunk 1]
```diff
@@ -26,8 +26,10 @@ def is_convertible(self, value: Any) -> bool:
         provider = self.network_manager.active_provider
         if not provider:
             raise ConversionError("Must be connected to a provider to use the token converter.")
-
-        tokens = self.manager.get_tokens(chain_id=provider.network.chain_id)
+        try:
+            tokens = self.manager.get_tokens(chain_id=provider.network.chain_id)
+        except:
```
Comment (unparalleled-js) [inline ape_tokens/converters.py]:
```suggestion
        except ValueError as err:
```

Let the other exceptions blow up for now.

Comment (Ninjagod1251) [inline ape_tokens/converters.py:31]: `except tokens.DoesNotExist:`

Comment (Ninjagod1251) [reply ape_tokens/converters.py:31]: dont add a blank expect and it should be okay

File: ape_tokens/converters.py [hunk 2]
```diff
@@ -26,8 +26,10 @@ def is_convertible(self, value: Any) -> bool:
...
+            return False
```
Comment (unparalleled-js) [inline ape_tokens/converters.py:32]: Above this line would be a good place to emit the warning:

`logger.warn_from_exception(err)`

NOTE: You will have to import the logger from the top of the module: `from ape.logging import logger`

File: setup.py [hunk 3]
```diff
@@ -11,7 +11,7 @@
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
     ],
     "lint": [
-        "black>=21.12b0,<22.0",  # auto-formatter and linter
+        "black>=22.3.0",  # auto-formatter and linter
```
Comment (unparalleled-js) [inline setup.py:14]:
```suggestion
        "black>=22.3.0,<23.0",  # auto-formatter and linter
```

**Review by unparalleled-js — COMMENTED** [review]
Looking gooooood

**Review by unparalleled-js — APPROVED** [review]
LGTM

**Pull request merged**